Crashlytics/Crashlytics/Controllers/FIRCLSReportUploader.m

```diff
@@ -60,7 +62,10 @@ - (instancetype)initWithManagerData:(FIRCLSManagerData *)managerData {
   _installIDModel = managerData.installIDModel;
   _fileManager = managerData.fileManager;
   _analytics = managerData.analytics;
-
+  _onDemandModel = [[FIRCLSOnDemandModel alloc]
+      initWithOnDemandUploadRate:[managerData.settings onDemandUploadRate]
+                            base:[managerData.settings onDemandBackoffBase]
+                    stepDuration:[managerData.settings onDemandBackoffStepDuration]];
```

Comment:
Nit: I think you can use `.onDemandUploadRate`, etc if you make these `@property(readonly)` and implement the getter

Reply:
Kindof a nit since we're not doing this for other settings

Reply:
done